cfn-resources/project/.rpdk-config

```diff
@@ -1,7 +1,7 @@
 {
     "typeName": "MongoDB::Atlas::Project",
     "language": "go",
-    "runtime": "go1.x",
+    "runtime": "provided.al2",
```

Comment:
if this works, should we change it in all resources?

Reply:
Yes, if it works I will change it for all resources.

Reply:
It worked well for project resource. I updated it for all other resources

Reply:
thx!